src/plugins/elasticsearch/index.js

```diff
@@ -24,7 +24,7 @@ module.exports = function (kibana) {
           key: Joi.string()
         }).default(),
         apiVersion: Joi.string().default('2.0'),
```

Comment:
That's a good question. Bumping that to 2.1 causes kibana to crash due to 2.1 not being considering a valid version.  I think we'd need to also bump our elasticsearch js dependency to accommodate that.